README.md

```
@@ -173,5 +226,213 @@ Handler -> UseCase -> Repository
+
+- 성공 시 `message`와 `data` 필드가 모두 포함됩니다:
+  ```json
+  {
+    "message": "success",
+    "data": [
+      /* 결과 데이터 */
+    ]
+  }
+  ```
+- 예시:
+  ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)